src/server/pps/server/api_server.go

```diff
@@ -703,13 +724,10 @@ func (a *apiServer) intersectCommitSets(ctx context.Context, commits []*pfs.Comm
 // ListJobStream.
 func (a *apiServer) listJob(
```

Comment:
I think that `listJob` has only one caller now; if so, you might as well merge it into `ListJob`.